client/src/app/videos/recommendations/recommended-videos.component.html

```diff
@@ -1,12 +1,20 @@
 <div class="other-videos">
   <ng-container *ngIf="hasVideos$ | async">
-    <div i18n class="title-page title-page-single">
-      Other videos
+    <div class="d-flex title-page-container">
+      <div i18n class="title-page title-page-single">
+        Other videos
+      </div>
+      <div *ngIf="!playlist" class="d-flex title-page-autoplay">
+        <span i18n>Autoplay</span>
+        <p-inputSwitch [(ngModel)]="autoPlayNextVideo" (ngModelChange)="switchAutoPlayNextVideo()"></p-inputSwitch>
+      </div>
     </div>
 
-    <div *ngFor="let video of (videos$ | async)">
+    <div *ngFor="let video of (videos$ | async); let i = index; let length = count">
       <my-video-miniature [video]="video" [user]="user" (videoBlacklisted)="onVideoRemoved()" (videoRemoved)="onVideoRemoved()">
       </my-video-miniature>
+      
+      <hr *ngIf="!playlist && i == 0 && length > 1" />
```

Comment:
If length === 0 this block should not be displayed (because there are no iteration in the ngfor)

Reply:
I'm only interested in the case where this is the first element of a list that contains more than one video recommended, and the current video played is not part of a playlist.

Reply:
Yep sorry :+1: